ansible/pbTestScripts/updateBoxes.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+
+FORCE=""
+REMOVE=false
+
+Usage(){
+  echo "
+Usage: ./updateBoxes.sh [options]
+
+Bash script to update vagrant boxes and remove old versions.
+
+Options:
+  --remove | -r[f]	Remove outdated boxes. ('-rf' will force remove outdated boxes)
```

Comment:
I presume `-f` just makes it noninteractive (doesn't prompt for user input)?

Reply:
Ref my other comment - perhaps "Remove outdated boxes instead of updating them" would clarify what happens in the default case.